src/connections/ResetPassword.ts

```diff
@@ -0,0 +1,82 @@
+import { createSelector } from "reselect";
+
+import { requestPasswordReset, resetPassword } from "@/generated/v3/userService/userServiceComponents";
+import {
+  requestPasswordResetFetchFailed,
+  requestPasswordResetIsFetching,
+  resetPasswordFetchFailed,
+  resetPasswordIsFetching
+} from "@/generated/v3/userService/userServicePredicates";
+import { ApiDataStore, PendingErrorState } from "@/store/apiSlice";
+import { Connection } from "@/types/connection";
+import { connectionHook } from "@/utils/connectionShortcuts";
+import { selectorCache } from "@/utils/selectorCache";
+
+export const RequestPasswordReset = (emailAddress: string, callbackUrl: string) =>
```

Comment:
exported functions should start with a lower case (`requestPasswordReset`). Starting with upper case is reserved for classes.